docs/configure-louie.md

```diff
@@ -0,0 +1,128 @@
+# Configure Louie
+## OAuth2 
+Louie serves as an OAuth2 client to Nexus. When Graphistry credentials are provided as environment variables, authentication is bypassed. To utilize a private Graphistry server, follow these steps: 
+ 
+- Register a client app in the Nexus admin panel: [OAuth2 Client Setup](https://github.com/graphistry/graphistry/wiki/OAuth2-Client-Setup) 
+- Configure OAuth2 settings in the  .env  file as per the  .env.example  
+ 
+If necessary, remove  lui_*  cookies using browser developer tools (navigate to Application -> Cookies). 
+ 
+## Use Louie with Local Nexus 
```

Comment:
Louie users would run Louie against Graphistry Hub or a self-hosted Louie AMI, on a separate server, so discussing Local Nexus seems confusing. That seems more appropriate for local dev docs for now.

Reply:
I think this section is intended to be `self-hosted graphistry` or `self-hosted louie` ? Given these specific scripts assume the additional ability to run nexus commands on a self-hosted graphistry node with commandline access, vs from anywhere via the secure REST api, that needs to be further qualified (and part of why that was not how these scripts were supposed to be written)